legend-engine-language-pure-modelManager-sdlc/src/main/java/org/finos/legend/engine/language/pure/modelManager/sdlc/SDLCLoader.java

```diff
@@ -180,7 +180,10 @@ else if (context.sdlcInfo instanceof AlloySDLC)
                     }
                     else
                     {
-                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths.toString() + " do not exist in the project data loaded from the metadata server");
+                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths.toString() + " do not exist in the project data loaded from the metadata server. " +
+                                "Please make sure the GitLab pipeline for a new release has completed and also metaData server has latest revision entities " +
```

Comment:
could we mention the version. 
"Please make sure the corresponding Gitlab pipeline for version x has dcompleted and also metaData server has updated with correspondingn entities."
If revisions are allowed you can mentioned the revision.

Reply:
dont think this line is necessary 
""GitLab pipeline running and metaData server job execution takes some minutes.. ""